src/v/cluster/feature_table.h

```diff
@@ -93,6 +94,12 @@ constexpr static std::array feature_schema{
     feature::maintenance_mode,
     feature_spec::available_policy::always,
     feature_spec::prepare_policy::always},
+  feature_spec{
+    cluster_version{3},
```

Comment:
@jcsp 

even though the `maintenance_mode` feature spec has a cluster_version=3 my reading was that I didn't need to bump the version for this new feature to cluster_version=4 unless bits on the wire change.

is that right?

Reply:
Yes, two features can be added in the same version as long as the version isn't already released, so at this point adding another feature to version 3 is fine.